clientui/src/hackathon/static/route.js

```diff
@@ -0,0 +1,384 @@
+// API Key for Mapbox
+mapboxgl.accessToken =
+  "pk.eyJ1Ijoic2Vhbi1tZWFkZSIsImEiOiJja3lmeDZkM3Iwc21hMm9xcG95YnFqaHh3In0.p4oU6PP7a92U1JYLBCLG2g";
```

Comment:
I reckon is best to have this access token in the `.yml` file - perhaps @gomyar can give his two cents on how to pull it here